### DISCUSSION TITLE
Why is GitHub not sending me daily newsletters anymore?

### DISCUSSION THREAD

**Discussion by sujjeee** [community/community#68575] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Previously, GitHub used to send me daily newsletters about what’s trending on GitHub and also recommend me repositories according to my interests. But now I don’t know why it stopped sending me daily newsletters about these things. I checked my email preferences and it still shows that I’m subscribed to it, but I’m not getting any newsletters. Why is that?

**Comment by Omar-Hosny1** [comment]


**Reply by queenofcorgis** [reply]
Hi @Omar-Hosny1 ,

Thanks so much for participating in the Community! While we encourage non-AI answers to questions in community, we also love experimenting and using new technologies. We ask if you do use ChatGPT or similar programs, please take the time to verify that this is the correct response and make it into your own authentic voice. 

ChatGPT answers in a tone that sounds like a tech support professional and while we support trying out this software, it is important to verify that they are solutions before posting otherwise it may mislead users into thinking that they are receiving official responses when they are not.

Per ChatGPT's website, the program does have limitations like:
- "May occasionally generate incorrect information"
- "May occasionally produce harmful instructions"
- "Limited knowledge of world and events after 2021"

**Comment by makumi10** [comment]
Check Your GitHub Notification Settings:

Go to your GitHub account settings by clicking on your profile picture in the top right corner and selecting "Settings."
In the left sidebar, click on "Notifications."
Ensure that your email notification preferences are correctly configured. Specifically, make sure that "Participating" and "@mentions" are set to "Email" and "On GitHub.
Sometimes, GitHub notification emails can end up in your spam or junk folder. Check these folders to see if the emails are being filtered there.
